frontend/app/src/components/Layout/AvatarMenu.tsx

```diff
@@ -22,41 +21,29 @@ export const AvatarMenu = ({ user }: AvatarMenuProps) => {
   const navigate = useNavigate();
 
   return (
-    <Menu>
+    <Menu variant="outline">
       <MenuButton
-        cursor="pointer"
         as={Avatar}
+        cursor="pointer"
         name={user.name}
         src={user.avatarUrl}
+        size="sm"
       />
-      {/* TODO: Abstract into theme components (for some reason doesn't work) */}
-      <MenuList bg="dark.500">
+      <MenuList>
         <MenuItem
-          icon={<Icon as={BiUser} />}
-          bg="dark.500"
-          _hover={{ bg: "dark.400" }}
-          color="light.100"
+          icon={<BiUser />}
```

Comment:
any reason you changed this? i followed the initial method cos of this
https://chakra-ui.com/docs/components/icon#using-a-third-party-icon-library

Reply:
Reverted in b87853f032290d01379ede40b5410de1336a0f56